apps/member-profile/app/shared/components/resource-form.tsx

```diff
@@ -158,6 +158,13 @@ export function ResourceTagsField({
   function reset() {
     setSearch('');
     setNewTagId(id());
+    listFetcher.submit(
+      { search: '' },
+      {
+        action: '/api/tags/search',
+        method: 'get',
+      }
+    );
```

Comment:
I like this solution! It's simple and resolves the bug. I do wonder if we can optimize further since we are passing in an empty string as the search parameter. Maybe we can replace this with a simpler 
```    
listFetcher.load('/api/tags/search');
```
which will default to a GET method with an empty string.

Reply:
Thanks! Let's try that out. I was messing around with this, goes to show I am a react newbie 🤭